governance/votes/Executive vote - September 27, 2023.md

```diff
@@ -0,0 +1,88 @@
+
+If this executive proposal passes, the Monetalis Clydesdale (RWA007-A) [RWA Agreement (doc)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) will be updated to [QmWo3UVtEDKVwS5k34uLt1J6u9px3rjHYkTLK2rYQ31E3G](https://jade-ltd-quail-880.mypinata.cloud/ipfs/QmWo3UVtEDKVwS5k34uLt1J6u9px3rjHYkTLK2rYQ31E3G).
+
+It includes the latest [DAO Resolution](https://amaranth-secondary-stingray-939.mypinata.cloud/ipfs/QmZ94FG8YXK4seyBHBi2FfTfW5URtBqbCb7JZAB1HGkTNF)
+
+This DAO Resolution will instruct the trustees of the Monetalis Clydesdale (RWA007-A) to the following JAT1 (James Asset Trust 1) to JAT2 (James Asset Trust 2) Asset Reallocation:
+- Selling the ETF products within JAT1.
+- Paying any proceeds from this sale after expenses in excess of 500 million USD into the surplus buffer.
+- Deploying the 500 million USD towards the T-bills ladder already installed under JAT2.
+
+### HV Bank (RWA009-A) Vault Doc Update
+
+*Authorization*: [Governance Executive 2023-09-13](https://vote.makerdao.com/executive/template-executive-vote-stability-scope-parameter-changes-spark-protocol-d3m-parameter-changes-set-fortunafi-debt-ceiling-to-zero-dai-dao-resolution-for-hv-bank-delegate-compensation-and-other-actions-september-13-2023)
+*Proposal*: [Forum post](https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/13) "[RWA009] HVBank/MIP21 Token CES Domain Team Assessment", [Forum post](https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/16) "Huntingdon Valley Bank: Transaction Documents on Permaweb"
+
+If this executive proposal passes, the HV Bank (RWA009-A) [RWA Agreement (doc)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) will be updated to [QmYjvAZEeGCs8kMuLQz6kU8PWgsbG1i8QWd2jrwkSipcRx](https://jade-ltd-quail-880.mypinata.cloud/ipfs/QmYjvAZEeGCs8kMuLQz6kU8PWgsbG1i8QWd2jrwkSipcRx).
+
+It includes the latest [DAO Resolution](https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/16)
+
+This resolution is intended to terminate future purchases into the RWA Master Participation Trust.
+
+### Fortunafi (RWA005-A) vault changes
+
+*Authorization*: [Governance Executive 2023-09-13](https://vote.makerdao.com/executive/template-executive-vote-stability-scope-parameter-changes-spark-protocol-d3m-parameter-changes-set-fortunafi-debt-ceiling-to-zero-dai-dao-resolution-for-hv-bank-delegate-compensation-and-other-actions-september-13-2023)
+*Proposal*: [Forum post](https://forum.makerdao.com/t/request-to-poll-offboarding-legacy-legal-recourse-assets/21582/12) "[Request to Poll] Offboarding Legacy Legal Recourse Assets"
+
+If this executive proposal passes, `tell` will be called on the RWA Liquidation Oracle for this vault type; this places the vault into a “soft liquidation”.
```

Comment:
We _just_ literally got the confirmation from Centrifuge, Dewiz and Steakhouse about recommending `tell` today.

So I recommend you set up the inclusion of this item by: updating the Forum thread with an explanation of why calling `tell` is being recommended and included in this Exec Vote. 

I think this is a good thing to clarify for the community bc there was a lot of confusion over this in the past.